src/main/java/minicraft/screen/SkinDisplay.java

```diff
@@ -214,7 +219,7 @@ public void render(Screen screen) {
 
 		// Help text.
 		Font.drawCentered("Use "+ Game.input.getMapping("cursor-down") + " and " + Game.input.getMapping("cursor-up") + " to move.", screen, Screen.h - 17, Color.DARK_GRAY);
-		Font.drawCentered(Game.input.getMapping("SELECT") + " to select, and " + Game.input.getMapping("EXIT") + " to cancel." , screen, Screen.h - 9, Color.DARK_GRAY);
+		Font.drawCentered(Game.input.getMapping("SELECT") + " to select, " + Game.input.getMapping("EXIT") + " to cancel." , screen, Screen.h - 9, Color.DARK_GRAY);
```

Comment:
This is gramatically incorrect.

Reply:
did not fit the screen

Reply:
Split it into two lines

Reply:
It’s for space reasons. Otherwise it goes off the screen.

Reply:
I encountered this too